lib/ext/heroku/command/addons.rb

```diff
@@ -203,7 +203,14 @@ def create
 
       action("Creating #{addon['name'].downcase}") {}
       action("Adding #{addon['name'].downcase} to #{app}") {}
-      action("Setting #{addon['config_vars'].join(', ')} and restarting #{app}") do
+
+      if !addon['config_vars'].empty?
+        message ="Setting #{addon['config_vars'].join(', ')} and restarting #{app}"
+      else
+        message = "Restarting #{app}"
```

Comment:
If the add-on doesn't provide any config vars, no release is created and no restart happens—we can leave this part out. :boom: